org-formation/600-access/_tasks.yaml

```diff
@@ -417,3 +417,35 @@ SynapseLlmDeveloperPolicy:
         ]
       }
     PolicyName: SynapseLlmDeveloperPolicy
+
+SynapseLlmLambdaExecRoleBoundary:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/IAM/managed-policy.yaml
+  StackName: synllm-lambdaexecrole-boundary
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+    Account:
+      - !Ref SynapseLlmProdAccount
+    Region: !Ref primaryRegion
+  Parameters:
+    PolicyDocument: >-
```

Comment:
please [read this article](https://easytocloud.com/index.php/2020/11/27/iam-permission-boundaries/), it implements a permission boundary for the exact same use case.  Take note of the boiler plate set of permissions you should add to the PB.  I think you need to add them as well..

The next step is to ensure inheritance of the policy. What we’re trying to achieve is the following:
* Any user or role that is created, needs to have this boundary policy attached. In other words, we deny API calls that create users or roles without this PB.
* You cannot attach another boundary policy to an existing user or role. In other words, we deny API calls that try to put a new PB on a user or role unless the PB is this PB.
* You cannot delete this boundary policy from an existing user or role

Reply:
yes, because you will still want the deny to be there in case  the `PowerUserAccess` policy is removed.